Dockerfile

```diff
@@ -1,9 +1,25 @@
-FROM scratch
+FROM golang:1.14-buster AS build
+
+ENV GOBIN=$GOPATH/bin
+
+ADD . /src/estafette-gke-preemptible-killer
+
+WORKDIR /src/estafette-gke-preemptible-killer
+
+RUN apt-get -qqq update \
+    && apt-get -qqq -y install ca-certificates\
+    && update-ca-certificates
+
+RUN go mod download \
```

Comment:
Although it's nice to be able to build the application inside the dockerfile so you can run all of it locally, I'd rather do this inside the separate stage in the `.estafette.yaml` file, because it gives better visibility inside the Estafette GUI of how much time it takes, etc.